Makefile

```diff
@@ -104,7 +106,12 @@ cover:
 xbuild: clean generate
 	@CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o $(NAME) cmd/faktory/daemon.go
 	# brew install upx
-	upx -qq ./faktory
+	upx -qq ./$(NAME)
+
+xbuild_arm64: clean generate
```

Comment:
This should not be a separate target. xbuild should build both.

Reply:
but they have the same name...
I can put them in a separte folder, perhaps a subfolder in the output folder?
Is cleaning between arch builds required?

Reply:
Arch subfolders seems fine.